bundles/managed-api-service/1.32.0/manifests/managed-api-service.clusterserviceversion.yaml

```diff
@@ -217,6 +221,12 @@ spec:
               verbs:
                 - create
                 - list
+            - apiGroups:
+                - ""
+              resources:
+                - pods/exec
+              verbs:
+                - create
```

Comment:
Missing permission in bundle csv from https://github.com/integr8ly/integreatly-operator/pull/3060